sqrl-planner/src/main/java/com/datasqrl/planner/graphql/GraphqlSchemaValidator.java

```diff
@@ -304,13 +304,21 @@ private InputObjectTypeDefinition getValidMutationInputType(
     return (InputObjectTypeDefinition) typeDef.get();
   }
 
+  private static Type unwrapNullAndList(Type type) {
+    for (; ; ) {
+      if (type instanceof NonNullType nullType) {
+        type = nullType.getType();
+      } else if (type instanceof ListType listType) {
+        type = listType.getType();
+      } else break;
+    }
+    return type;
```

Comment:
wondering if we could hit some bug that would make this hang....

maybe make it recursive, at least it would exit on stackoverflow

```
private static Type unwrapNullAndList(Type type) {
  if (type instanceof NonNullType nullType) {
    return unwrapNullAndList(nullType.getType());
  } else if (type instanceof ListType listType) {
    return unwrapNullAndList(listType.getType());
  } else {
    return type;
  }
}

```